sdk/synapse/Azure.Analytics.Synapse.ManagedPrivateEndpoints/src/Azure.Analytics.Synapse.ManagedPrivateEndpoints.csproj

```diff
@@ -38,5 +37,8 @@
     <Compile Include="$(AzureCoreSharedSources)TaskExtensions.cs" LinkBase="Shared" />
     <Compile Include="$(AzureCoreSharedSources)AzureResourceProviderNamespaceAttribute.cs" LinkBase="Shared" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Let's keep the package reference in this one, as it's not yet an LLC library.

Reply:
Now that we have added the Experimental condition in Generator.Shared code, I added the package reference back and removed the project reference.